tasks/install.yml

```diff
@@ -0,0 +1,68 @@
+---
+- import_tasks: apt_install.yml
+  when: ansible_pkg_mgr == "apt"
+
+- name: Add gitlab user
```

Comment:
*GitLab. You actually did it correct in the comment

Reply:
fixed